tests/MongoDB.Bson.Tests/TargetFrameworkTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using FluentAssertions;
+using Xunit;
+
+namespace MongoDB.Bson.Tests
+{
+    public class TargetFrameworkTests
+    {
+        [Fact]
+        public void TargetFramework_should_be_valid()
+        {
+            var actual = MongoDB.Bson.TargetFramework.Moniker;
```

Comment:
The suggestion was to use `actualFramework` and `expectedFramework`.

We don't usually use variable names as vague as `actual` and `expected`.

What do you  think?